src/elastio-snap.c

```diff
@@ -5234,7 +5259,11 @@ static void __tracer_unverified_snap_to_active(struct snap_device *dev, const ch
 	if(ret) goto error;
 
 	//setup the cow manager
-	ret = __tracer_setup_cow_reload_snap(dev, dev->sd_base_dev, cow_path, dev->sd_size, dev->sd_cache_size);
+	if(is_pathname_exists(cow_path)){
+		ret = __tracer_setup_cow_reload_snap(dev, dev->sd_base_dev, cow_path, dev->sd_size, dev->sd_cache_size);
+	}else{
+		ret = __tracer_setup_cow_reload_snap(dev, dev->sd_base_dev, rel_path, dev->sd_size, dev->sd_cache_size);
+	}
```

Comment:
Does it make sense to move this logic inside of `__tracer_setup_cow` func?
Maybe it'll allow to simplify code here and [here](https://github.com/elastio/elastio-snap/pull/198/files#diff-3d1684da11d36e060b94eb1385661e3c5d38497cfaf23118df4199f410ba3593R5328-R5332) and [here](https://github.com/elastio/elastio-snap/pull/198/files#diff-3d1684da11d36e060b94eb1385661e3c5d38497cfaf23118df4199f410ba3593R5367-R5379) ...